apps/coscrad-frontend/src/utils/generic-components/presenters/multilingual-text-presenter.tsx

```diff
@@ -44,17 +44,6 @@ export const MultilingualTextPresenter = ({
                             </Tooltip>
                         )}
                     </Typography>
-                    {isNullOrUndefined(textItemWithDefaultLanguage) ? null : (
-                        <Tooltip
-                            title={`${getLabelForLanguage(
-                                textItemWithDefaultLanguage.languageCode
-                            )}, '${textItemWithDefaultLanguage.role}'`}
-                        >
-                            <IconButton>
-                                <LanguageIcon />
-                            </IconButton>
-                        </Tooltip>
-                    )}
                 </AccordionSummary>
```

Comment:
I'd like to see this in action. I think this has been causing some trouble in some places until now. Be sure to run all Cypress tests, as there are some tests where we had to manually expand the multilingual text presenter to find the translation text. If that behaviour has changed, you'll need to update the corresponding tests.

Reply:
I think it's important to have the multilingual text presenter show some multilingual text rather than "Translations".  In my build with the data I have and default language for multilingual presenter set to Haida, the first thing I see is:
<img width="646" alt="image" src="https://github.com/COSCRAD/coscrad/assets/63014783/05a5bbaa-a296-4b38-8614-7edf0551cbc7">
And then, when I open the accordion I see:
<img width="613" alt="image" src="https://github.com/COSCRAD/coscrad/assets/63014783/62dd6636-9dc4-4c3b-bdb4-e8203ed326ae">
But what I'd like to see is "The Demonstration".  ie, I would make sense to me that the multilingual text should run through the multilingual text items to find the best available option our users have entered to display, rather than just "Translations".